config/environments/production.rb

```diff
@@ -78,21 +78,35 @@
 
   if ENV['MAILTRAP_ENABLED'] == 'enabled'
     config.action_mailer.delivery_method = :mailtrap
+  elsif ENV['MAILCATCHER_ENABLED'] == 'enabled'
+    config.action_mailer.delivery_method = :mailcatcher
+  else
+
+    sendinblue_weigth = case [ENV['SENDINBLUE_ENABLED'] == 'enabled', ENV['SENDINBLUE_BALANCING'] == 'enabled']
+      in [false, _]
+        0
+      in [true, false]
+        100
```

Comment:
je ne connaissais pas cette synthaxe

Reply:
ruby 3.1 , pattern matching for the win https://www.toptal.com/ruby/ruby-pattern-matching-tutorial